src/cobol.py

```diff
@@ -0,0 +1,21 @@
+import sys
+
+from antlr4 import *
+
+from Cobol85Lexer import Cobol85Lexer
+from Cobol85Listener import Cobol85Listener
+from Cobol85Parser import Cobol85Parser
+
+
+def main(program_file):
+    lexer = Cobol85Lexer(FileStream(program_file))
+    tokens = CommonTokenStream(lexer)
+    parser = Cobol85Parser(tokens)
+    tree = parser.compilationUnit()
+
+    extractor = Cobol85Listener()
```

Comment:
the listener is what applies the custom logic to pull interesting data (data definition statements) out of the parsed tree.